ingestion-beam/src/test/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRowTest.java

```diff
@@ -123,6 +123,19 @@ public void testAdditionalProperties() throws Exception {
     assertEquals(expectedAdditional, Json.asString(additionalProperties));
   }
 
+  @Test
+  public void testStrictSchema() throws Exception {
+    Map<String, Object> parent = new HashMap<>();
+    parent.put("outer", new HashMap<>(ImmutableMap.of("otherfield", 3, //
```

Comment:
Might be good to add a telemetry test now.

Reply:
Added an integration test that exercises `bqStrictSchemaDocTypes`